quickstarts/microsoft.search/azure-search-create-private-endpoint/README.md

```diff
@@ -0,0 +1,10 @@
+# Azure Cognitive Search service
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.search%2Fazure-search-create-private-endpoint%2Fazuredeploy.json)
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.search%2Fazure-search-create-private-endpoint%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fquickstarts%2Fmicrosoft.search%2Fazure-search-create-private-endpoint%2Fazuredeploy.json)
+
+This template creates a new Azure Cognitive Search Service with a private endpoint. The [quickstart](https://docs.microsoft.com/azure/search/search-get-started-arm) article describes how to deploy the template.
+
+If you are unclear as to what parameters are allowed you can check the [Microsoft.Search searchServices template reference](https://docs.microsoft.com/azure/templates/Microsoft.Search/searchServices).
```

Comment:
For more information about parameters, see [Microsoft.Search searchServices template reference](/azure/templates/Microsoft.Search/searchServices).